sections/futures/Trade/CrossMarginAccountActions.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import styled from 'styled-components';
+
+import Button from 'components/Button';
+import useGetCrossMarginAccountOverview from 'queries/futures/useGetCrossMarginAccountOverview';
+import { zeroBN } from 'utils/formatters/number';
+
+import CrossMarginOnboard from '../CrossMarginOnboard';
+
+const CrossMarginAccountActions: React.FC = () => {
+	const { t } = useTranslation();
+	const [openModal, setOpenModal] = React.useState<'deposit' | 'withdraw' | null>(null);
+
+	const query = useGetCrossMarginAccountOverview();
+	const freeMargin = query.data?.freeMargin || zeroBN;
+
+	return (
+		<>
+			<MarketActionsContainer>
+				<MarketActionButton onClick={() => setOpenModal('deposit')} noOutline>
+					{t('futures.market.trade.button.deposit')}
+				</MarketActionButton>
+				<MarketActionButton
+					disabled={freeMargin.lte(zeroBN)}
```

Comment:
This state doesn't seem to add any styling. Clicking does nothing, but I still get a hand cursor and hover state for the button.

Reply:
It was temporary UI so going to leave for now as we have new designs to follow for these actions.